data/mods/Generic_Guns/modinfo.json

```diff
@@ -4,7 +4,7 @@
     "id": "generic_guns",
     "name": "Generic Guns",
     "authors": [ "Tonkatsu" ],
-    "maintainers": [ "Tonkatsu" ],
+    "maintainers": [ "" ],
```

Comment:
```suggestion
    "maintainers": [ "CleverRaven" ],
```

Reply:
if @kevingranade doesn't object

Reply:
Committing the suggestion, though its on whichever merger to assign this responsibility to the organization.